docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -114,33 +133,32 @@ Play is being rebuilt and progressively rolled out to more users. This section e
 For Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
 Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
 
-For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag. It is `true` by default for the Docker distribution for development and `false` by default on the Kubernetes distribution for production use.
-
-Play uses Zeebe 8.2. Any BPMN elements unavailable in Zeebe 8.2, such as signal events, will not be available in Play.
+For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag. It is `true` by default for the Docker and Kubernetes distributions.
```

Comment:
Even Web Modeler is enabled, you would still need to pass PLAY_ENABLED flag to the webapp image, which wash the issue about. I don't think we would have to enable Web Modeler by default as well, right? But its important Play is there by default, in case Web Modeler starts

Reply:
@huygur No we should not enable web modeler by default. I would simply add something like "[...] is `true` by default for the Docker and Kubernetes distributions. Thus, if you enable Web Modeler, Play will be enabled automatically"
I think it's always good to add context. reading this by itself one could think that play will be accessible with the default config. which it's not, as web modeler isn't enabled